dist/analytics.js

```diff
@@ -152,9 +152,10 @@ function safelyCaptureMessage(message, level) {
         Sentry.captureMessage(message, level);
     }
 }
-function buttonClickedEvent() {
+function buttonClickedEvent(eventNameOverride) {
+    if (eventNameOverride === void 0) { eventNameOverride = null; }
```

Comment:
Always funny to see the syntax the Typescript compiler ends up choosing. Way less intuitive :-D